### DISCUSSION TITLE
Small feedback - making `legend` on visual-picker optional

### DISCUSSION THREAD

**Discussion by NolanChan** [twilio-labs/paste#3481] [General] [answered]
We have a use case where we wanted to use the visual-picker for selecting a destination, but it was part of a larger component and didn't need the `legend`. I ended up hardcoding the `legend` attribute to be an empty string, but just wanted to put that feedback out there

**Comment by SiTaggart** [comment, marked as answer]
Hi there

As per [the documentation](https://paste.twilio.design/components/visual-picker#accessibility) the Legend isn't optional for Accessibility requirements.

> Like all input elements, both Radio and Checkbox Visual Pickers require a group label that describes the collection. Provide an accessible label using the legend prop on VisualPickerRadioGroup or VisualPickerCheckboxGroup. 

It's an important part of the component, and without it, creates a less than ideal experience for users.
